tests/check/check.go

```diff
@@ -1368,6 +1368,6 @@ func assertListUsers(ctx context.Context, t *testing.T, assertion *checktest.Ass
 	}
 }
 
-func runTestMatrixSuite(t *testing.T, client ClientInterface) {
+func RunTestMatrixSuite(t *testing.T, client ClientInterface) {
```

Comment:
This allows other projects that use OpenFGA as library to be able to run test matrix suite for their project's datastore.